server/plugin/flows.go

```diff
@@ -0,0 +1,763 @@
+					Title:       "Create Webhook",
+					SubmitLabel: "Create",
+					Elements: []model.DialogElement{
+						{
+
+							DisplayName: "", // TODO: This will still show a *
+							Name:        "repo_org",
+							Type:        "text",
+							SubType:     "text",
+							Placeholder: "Repository or organization name, e.g. mattermost/mattermost-server",
+							HelpText:    "Specify repository or organization for which you want to create a webhook",
+						},
+					},
+				},
+				OnDialogSubmit: fm.submitWebhook,
+			},
+			OnClick: func(userID string) int {
+				return -1
+			},
+		}).
+		WithButton(steps.Button{
+			Name:  "No",
+			Style: steps.Default,
+			OnClick: func(userID string) int {
+				return 1
+			},
+		}).
+		WithButton(steps.Button{
+			Name:    "Cancel setup",
+			Style:   steps.Danger,
+			OnClick: fm.cancelFlow,
+		}).
+		Build()
+
+	confirmationStep := steps.NewEmptyStep("success", "Success! :tada: You've successfully set up your Mattermost GitHub integration! ", "Use `/github subscriptions add` to subscribe any Mattermost channel to your GitHub repository. [Learn more](https://example.org)")
+
+	steps := []steps.Step{
+		questionStep,
+		confirmationStep,
+	}
+
+	return steps
+}
+
+func (fm *FlowManager) submitWebhook(userID string, submission map[string]interface{}) (int, *steps.Attachment, string, map[string]string) {
+	repoOrgRaw, ok := submission["repo_org"]
+	if !ok {
+		return 0, nil, "repo_org missing", nil
+	}
+	repoOrg, ok := repoOrgRaw.(string)
+	if !ok {
+		return 0, nil, "repo_org is not a string", nil
+	}
+
+	org, repo := parseOwnerAndRepo(repoOrg, fm.getConfiguration().getBaseURL())
+	if org == "" && repo == "" {
+		return 0, nil, "Invalid format", nil
+	}
+
+	webhookEvents := []string{"create", "delete", "issue_comment", "issues", "pull_request", "pull_request_review", "pull_request_review_comment", "push", "star"}
+
+	config := map[string]interface{}{
+		"content_type": "json",
+		"insecure_ssl": "0",
+		"secret":       fm.getConfiguration().WebhookSecret,
+		"url":          fmt.Sprintf("%s/webhook", fm.pluginURL),
+	}
+
+	hook := &github.Hook{
+		Events: webhookEvents,
+		Config: config,
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), 28*time.Second) // HTTP request times out after 30 seconds
+	defer cancel()
+
+	client, apiErr := fm.getGitHubClient(ctx, userID)
+	if apiErr != nil {
+		return 0, nil, apiErr.Error(), nil
+	}
+
+	ch := fm.pingBroker.SubscribePings()
+
+	var err error
+	var resp *github.Response
+	var fullName string
+	if repo == "" {
+		fullName = org
+		hook, resp, err = client.Organizations.CreateHook(ctx, org, hook)
+	} else {
+		fullName = org + "/" + repo
+		hook, resp, err = client.Repositories.CreateHook(ctx, org, repo, hook)
+	}
+
+	if resp.StatusCode == http.StatusNotFound {
+		return 0, nil, fmt.Sprintf("It seems like you don't have access %s. Please ask an administrator of that repository to run /github setup webhook for you.", fullName), nil
```

Comment:
```suggestion
		return 0, nil, fmt.Sprintf("It seems like you don't have access %s. Ask a system admin of that repository to run `/github setup` webhook for you.", fullName), nil
```

Reply:
No support for Markdown in the error message :confused: